.gitignore

```diff
@@ -16,3 +16,6 @@
 
 # Jetbrains files
 .idea/
+
+# kiota
+kiota/
```

Comment:
I'm curious to why this is part of the gitignore?

Reply:
ah because you're cloning the kiota repo under that structure?

Reply:
yap

Reply:
alright, this will resolve itself when we close this PR in favor to the one in kiota.
FYI I either use submodules (when no other references options are available) or relative paths (outside of the repo) to avoid this kind of confusion.